### PR TITLE
Defined encoding to utf-8 which was causing decode error in windows

### LINKED ISSUE #978: open('docs/introduction.rst').read()) and UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 10312: character maps to <undefined>
When I try to install a package from GitHub, it gives me an error:

```
ERROR: Command errored out with exit status 1:
     command: 'D:\apps\Anaconda3\envs\autospider\python.exe' -c 'import sys, setuptools, tok
enize; sys.argv[0] = '"'"'C:\\Users\\AppData\\Local\\Temp\\pip-req-build-mvfy06ye\\se
tup.py'"'"'; __file__='"'"'C:\\Users\\AppData\\Local\\Temp\\pip-req-build-mvfy06ye\\s
etup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'
\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info -
-egg-base 'C:\Users\barto\AppData\Local\Temp\pip-pip-egg-info-i5jufi9v'
         cwd: C:\Users\barto\AppData\Local\Temp\pip-req-build-mvfy06ye\
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\AppData\Local\Temp\pip-req-build-mvfy06ye\setup.py", line 6, in <
module>
        introduction = re.sub(r':members:.+|..\sautomodule::.+|:class:|:func:|:ref:', '', op
en('docs/introduction.rst').read())
      File "\apps\Anaconda3\envs\lib\encodings\cp1250.py", line 23, in decode
        return codecs.charmap_decode(input,self.errors,decoding_table)[0]
    UnicodeDecodeError: 'charmap' codec can't decode byte 0x81 in position 10312: character
maps to <undefined>
```

The problem is with introduction.rst file. Changing `open('docs/introduction.rst').read())` in my fork repo into `open('docs/introduction.rst', encoding='utf-8').read())` in setup.py solved problem.

### PR DESCRIPTION
**Pull request by atharmohammad**
Closes #978

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
                         open('dateparser/__init__.py').read()).group(1)
 
 introduction = re.sub(r':members:.+|..\sautomodule::.+|:class:|:func:|:ref:',
-                      '', open('docs/introduction.rst').read())
+                      '', open('docs/introduction.rst', encoding='utf-8').read())
 history = re.sub(r':mod:|:class:|:func:', '', open('HISTORY.rst').read())
```
Comment (icanhasmath) [inline setup.py:9]: I believe you will also need to add utf-8 encoding for `HISTORY.rst` as well.

Comment (atharmohammad) [reply setup.py]: Yes! you are correct. Thanks :smile:

**Review by Gallaecio — APPROVED** [review]

**Review by lopuhin — COMMENTED** [review]
Thanks for PR @atharmohammad and sorry for a late review. It looks great, but in e14655bf18636624fd7827cb6ab5428ef4f007c0 the line endings in setup.py are changed, so the diff is showing that the whole file is changed (the line endings don't show up on github but are visible in the command line git). Would you mind fixing this?

**Comment by atharmohammad** [comment]
> Thanks for PR @atharmohammad and sorry for a late review. It looks great, but in [e14655b](https://github.com/scrapinghub/dateparser/commit/e14655bf18636624fd7827cb6ab5428ef4f007c0) the line endings in setup.py are changed, so the diff is showing that the whole file is changed (the line endings don't show up on github but are visible in the command line git). Would you mind fixing this?

Yeah! Sure. Fixing it

**Comment by atharmohammad** [comment]
@lopuhin  have done the changes please have a look !

**Review by lopuhin — APPROVED** [review]
Looks great, thanks @atharmohammad :+1:

**Pull request merged**